README.rst

```diff
@@ -1,5 +1,11 @@
 Python Client for Cloud AutoML API
 ==================================
+.. note::
```

Comment:
If we can't find a suitable design for this being in a note sort of type, could you move this below the badges?

Reply:
yep done